docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -531,16 +513,13 @@ spec:
 EOF
 ----
 
-* Adoption of the neutron-ovn-metadata-agent:
-+
-Neutron-ovn-metadata-agent running on the data plane nodes don't require any
-additional actions nor config adjustments to do during the adoption process.
-When OpenStackDataPlaneDeployment and OpenStackDataPlaneNodeSet will be ready,
-neutron-ovn-metadata-agent should be up and running on the data plane nodes.
+[NOTE]
+`Neutron-ovn-metadata-agent` running on the data plane nodes do not require any additional actions or configuration during adoption.
```

Comment:
I wonder if `does` should be used instead of `do`. The subject is `Neutron-ovn-metadata-agent` which is singular.